src/core/HTTP/RestClient.ts

```diff
@@ -62,8 +63,9 @@ export default class RestClient {
     this.client = axios.create({
       baseURL: this.clientUrl,
       headers: this.clientHeaders,
-      httpAgent: new HTTPAgent({ keepAlive: true }),
-      maxContentLength: 10 * 1000000, // 10mb is kong file system
+      httpAgent: new HTTPAgent({ keepAlive: true, maxSockets: 10 }),
```

Comment:
```suggestion
      httpAgent: new HTTPAgent({ keepAlive: true }),
```

Reply:
Yup apologies seemed to have missed that. I have now removed it in my latest commit